integration/src/test/resources/properties/language.yaml

```diff
@@ -0,0 +1,2 @@
+language:
+  pattern: "#,##0.00"
```

Comment:
this is not being actually used, and it may lead to confusion if it is here like this. Remove all content from this file and put a comment if this is needed anyway explaining why.